admittance_controller/include/admittance_controller/admittance_rule_impl.hpp

```diff
@@ -51,7 +53,7 @@ controller_interface::return_type AdmittanceRule::configure(
       kinematics_ = std::unique_ptr<kinematics_interface::KinematicsInterface>(
         kinematics_loader_->createUnmanagedInstance(parameters_.kinematics.plugin_name));
       if (!kinematics_->initialize(
-            node->get_node_parameters_interface(), parameters_.kinematics.tip))
+            node->get_node_parameters_interface(), parameters_.kinematics.tip, robot_description))
```

Comment:
```suggestion
            node->get_node_parameters_interface(), "kinematics", robot_description))
```

Propose to set here `param_base_name` so that we search for the prameters under `controller interface.kinematics`. As we are doing in [JointLimtiers](https://github.com/ros-controls/ros2_control/blob/master/joint_limits/include/joint_limits/joint_limits_rosparam.hpp)